### DISCUSSION TITLE
VectorScope and ElZone "false color" map for RV

### DISCUSSION THREAD

**Discussion by BunnyWooped** [AcademySoftwareFoundation/OpenRV#333] [Ideas]
Would there be an interest for that kind of debugging tools in RV?
If you don't know what it is (I'm sure, it's the opposite), these are just tools for exposure, color balance, saturation, etc.

**Comment by compaga** [comment]
Hi BunnyWooped, I am happy to see your interest in Open RV. There are already few of these in the Open RV "Color" menu. Exposure and Saturation for instance. Did you have something different in mind?

**Comment by BunnyWooped** [comment]
Well, yes.
El Zone
https://www.panasonicvisualsystems.com/wp-content/uploads/2021/04/EL-Zone-ON.jpg
VectorScope with skin tone indicator
https://timeinpixels.com/wp-content/uploads/2020/07/03_vectorscopesplit-1024x530.jpg

These are tools used in film making and I think, these would be usefull in RV for CGI previews.

**Reply by compaga** [reply]
Great! I will let others express their interest but it looks like it could be a good addition. You may also post in https://academysoftwarefdn.slack.com/archives/C01CC7EGCDB to get more opinions. Also, we are also trying to figure how the community could contribute packages/plugins to Open RV or other projects. We would really love to facilitate sharing plugins but we also want to keep this manageable.
